app/pods/publications/overview/urgent/route.js

```diff
@@ -9,20 +22,28 @@ export default class PublicationsOverviewUrgentRoute extends PublicationsOvervie
     'numberOfPages',
     'publicationDueDate',
   ];
-  tableConfigStorageKey = "publication-table.urgent";
+  tableConfigStorageKey = 'publication-table.urgent';
 
   beforeModel() {
-    super.beforeModel(...arguments)
+    super.beforeModel(...arguments);
+    const notPublishedStatuses = this.store
+      .peekAll('publication-status')
+      .filter((status) => {
```

Comment:
It’s maybe easier to understand the intention of the filter by just [rejecting](https://api.emberjs.com/ember/3.28/classes/EmberArray/methods/reject?anchor=rejectBy) the status where `status.isPublished` is true?

Reply:
Never used rejectBy before, didn't know it existed.
I probably could have reverse filtered with a normal filter t.b.h., I copy pasted the filtering from different route.

This looks way better though.
Thanks!